src/style.css

```diff
@@ -1 +1,132 @@
 /* add styles here */
+@font-face {
+  font-family: Avenir-Book, sans-serif;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir-Heavy, sans-serif;
```

Comment:
Similar to the previous comment, the 'font-family' should only specify the custom font name 'Avenir-Heavy' without 'sans-serif'.